views/layouts/main.handlebars

```diff
@@ -87,7 +94,7 @@
     </div>
   </div>
   {{> footer}}
-
+<script src="public/js/myScripts.js"></script>
```

Comment:
Hi @azharakbar, could you name the `js` file more readable? Something like `spinner.js` would be good

Reply:
@wildan3105 
ook ... sure :) 
I will send the PR within 5 minute :)

Reply:
I have corrected and raised the PR :) 
@wildan3105 Any more changes ?

Reply:
Hi @wildan3105 , I have made the changes ... :) And raised the new PR .. :)